src/main/drivers/vtx_rtc6705.c

```diff
@@ -97,28 +97,51 @@ static uint32_t reverse32(uint32_t in)
 /**
  * Start chip if available
  */
-void rtc6705IOInit(void)
+bool rtc6705IOInit(const vtxIOConfig_t *vtxIOConfig)
 {
+    busdev = &busInstance;
+
+    busdev->busdev_u.spi.csnPin = IOGetByTag(vtxIOConfig->csTag);
+
+    bool rtc6705HaveRequiredResources =
+        (busdev->busdev_u.spi.csnPin != IO_NONE);
+
 #ifdef RTC6705_POWER_PIN
-    vtxPowerPin = IOGetByTag(IO_TAG(RTC6705_POWER_PIN));
-    IOInit(vtxPowerPin, OWNER_VTX, 0);
+    vtxPowerPin = IOGetByTag(vtxIOConfig->powerTag);
 
-    DISABLE_VTX_POWER();
-    IOConfigGPIO(vtxPowerPin, IOCFG_OUT_PP);
+    rtc6705HaveRequiredResources &= (vtxPowerPin != IO_NONE);
```

Comment:
but it is probably bad idea, maybe it may be evaluated as `a=(bool)(a&b)` which will fail
`rtc6705HaveRequiredResources &= (bool)vtxPowerPin;` will work
`rtc6705HaveRequiredResources = rtc6705HaveRequiredResources && vtxPowerPin;` is probably cleanest solution

Reply:
yeah, you know I did questions myself when I used the `&=` operator on that boolean and almost did what you suggested but decided to use braces instead to make it clear there's an expression, I will do as you suggest.

Reply:
done